### ISSUE TITLE
bug with SS4, Elemental and UserForms

### ISSUE THREAD

**Issue by digitall-it** [silverstripe/silverstripe-subsites#343]
When in the Subsites panel, after creating a subsite if you switch to it from the dropdown it reports an error similar to `This form should be sent in POST`.

Here's my composer dependancies.
```
        "php": ">=5.6.0",
        "silverstripe/recipe-plugin": "^1",
        "silverstripe/recipe-cms": "1.0.3@stable",
        "silverstripe-themes/simple": "~3.2.0",
        "silverstripe/subsites": "dev-master",
        "silverstripe/userforms": "dev-master",
        "dnadesign/silverstripe-elemental": "dev-master",
        "dnadesign/silverstripe-elemental-subsites": "dev-master",
        "dnadesign/silverstripe-elemental-userforms": "^1.0@dev",
        "dnadesign/silverstripe-elemental-virtual": "^1.0@dev",
        "silverstripe/elemental-blocks": "^1.0@dev",
        "silverstripe/fulltextsearch": "dev-master",
        "dnadesign/silverstripe-elemental-list": "^1.0@dev"
```

**Comment by robbieaverill** [comment]
Reproduced, thanks for reporting

**Comment by robbieaverill** [comment]
Fix at #344
